scripts/check_text.py

```diff
@@ -31,11 +31,19 @@ def run():
     with open("bot/data/text/en.json", "r", encoding="utf8") as f:
         en_data = json.load(f)["en"]
 
+    with open("bot/secrets.json", "r") as f:
+        secrets = json.load(f)
+
+    disabled_translations = secrets.get("disabled_translations", [])
```

Comment:
Yknow now that I think about it, disabled_translations should probably go in `data.json` instead. That way it's present when `check_text` is ran.